share/mk/bsd.cpu.mk

```diff
@@ -348,13 +348,8 @@ CFLAGS+=	-Werror=implicit-function-declaration
 # XXXBD: is this still needed?
 CFLAGS+=	-Wno-deprecated-declarations
 # XXXBD: is -mstack-alignment needed here?
-. if ${MACHINE_ARCH:Mmips*c128}
 CFLAGS+=	-cheri=128
 CFLAGS+=	-mstack-alignment=16
```

Comment:
We can drop this too (and the comment just above); n64 requires 16-byte quadword alignment, only CHERI-256 ever needed to change the ABI there.